doc/content/devices/the-things-uno/_index.md

```diff
@@ -0,0 +1,432 @@
+
+Once uploaded, your sketch will immediately start to run but you cannot see any output unless you open the Arduino **Serial Monitor**. The Arduino Serial Monitor is a tool that can be used to print information passing through the serial port. 
+
+In the **menu bar** select **Tools &#8594; Serial Monitor**. Once open the Serial Monitor prints something similar to the following output.
+
+```
+-- STATUS
+EUI: 0004A30B001B7AD2
+Battery: 3223
+AppEUI: 70B3D57EF000001C
+DevEUI: 0004A30B001B7AD2
+Band: 868
+Data Rate: 5
+RX Delay 1: 1000
+RX Delay 2: 2000
+Total airtime: 0.00 s
+```
+
+**Copy** the Serial Monitor output to a text editor (i.e. Notepad) because you will need the **AppEUI** and **DevEUI** when you register your Things Node with the Things Stack.
+
+## Registering The Things UNO with The Things Stack
+It’s time to register your Things Uno with the Things Stack. 
+
+On the **Applications** page, select your application to go to its overview page. 
+
+Select **+ Add end device** in the bottom-right of the page.
+
+{{< figure src="AddEndDevice.png" alt="Add end device" >}}
+
+The Things Stack provides two options to register your end device.
+- From the LoRaWAN Device Repository
+- Manually
+
+Each registering option provides both **OTAA** and **ABP** activation methods. Let’s have a look at them one by one.
+
+### From the LoRaWAN Device Repository 
+This is the easiest way to register your Things Uno with the Things Stack.
+
+On the **Register end device** page, in the **From the LoRaWAN Device Repository** tab, under **Select the end device**, select the following mandatory fields:
+- Brand - `The Things Products`
+- Model – `The Things Uno`
+- Hardware Ver. – `1.0`
+- Firmware Ver. – `quickstart` or `abp` (we recommend you to use `quickstart` as it provides **OTAA**.)
+- Profile (Region) – Choose `EU_863_870` or `US_902_928` to match with your board. 
+
+{{< figure src="RegisterEndDeviceRepo.png" alt="Register End Device from LoRaWAN repository" >}}
+
+Under the **Enter registration data**, select/fill the following mandatory fields.
+
+If you have selected `quickstart` from the **Firmware Ver.** in the previous step:
+- Frequency plan – Select `Europe 863-870 MHz (SF9 for RX2 - recommended)` for **EU_863_870** or `United States 902-928 MHz, FSB2 (used by TTN)` for **US_902_928**.
+- AppEUI – Copy the **AppEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- DevEUI – Copy the **DevEUI** from the output printed by the ***TheThingsUnoTest*** sketch.
+- AppKey – Select **Generate** button to generate an **AppKey**.
+- End device ID – A unique human-readable identifier for your Things Uno.
+
+{{< note >}}
```

Comment:
Don't copy and paste this here. Use a link, i.e {{< ref "reference/id-eui-constraints" >}}

Reply:
done